crypto/bls12381_utils.c

```diff
@@ -442,6 +442,7 @@ int bn_read_Zr_bin(bn_t a, const uint8_t *bin, int len) {
     if (len!=Fr_BYTES) {
         return RLC_ERR;
     }
+    bn_new(a);
```

Comment:
Same comment here, it would be good to make the caller variables are initialized, as in the suggestion branch.